### PR TITLE
use meta to determine installer version

### LINKED ISSUE #14: Get installer from meta instead of Github Artifacts.

### PR DESCRIPTION
**Pull request by BluCobalt**
closes #14

### CODE REVIEW

**Comment by thecatcore** [comment]
@BleachDev can you take a look at this?

File: downloads.html [hunk 1]
```diff
@@ -32,10 +32,31 @@
...
-            <a href="https://maven.legacyfabric.net/net/legacyfabric/fabric-installer/0.11.1/fabric-installer-0.11.1.jar"><button class="download-button">Download .jar <br> (universal)</button></a>
+            <a href="https://maven.legacyfabric.net/net/legacyfabric/fabric-installer/0.11.1/fabric-installer-0.11.1.exe" id="installer-exe"><button class="download-button">Download .exe <br> (for windows)</button></a>
+            <a href="https://maven.legacyfabric.net/net/legacyfabric/fabric-installer/0.11.1/fabric-installer-0.11.1.jar" id="installer-jar"><button class="download-button">Download .jar <br> (universal)</button></a>
             <a href="https://github.com/Legacy-Fabric/multimc-instance-creator/releases"><button class="download-button">MultiMC <br> instances</button></a>
 </body>
+<script>
+    var fallback = "0.11.1"
+    function setLatestMeta() {
+        var latest
+        fetch("https://meta.legacyfabric.net/v2/versions/installer")
```
Comment (BoogieMonster1O1) [inline downloads.html:43]: this breaks internet explorer

**Review by BoogieMonster1O1 — APPROVED** [review]

**Review by BleachDev — APPROVED** [review]

File: downloads.html (hunk 1 above)
Comment (thecatcore) [reply downloads.html:43]: Amazing

**Review by thecatcore — APPROVED** [review]

File: downloads.html (hunk 1 above)
Comment (BluCobalt) [reply downloads.html:43]: if a browser doesn't support the javascript, it's still hardcoded in anyways so it'll still work even if its not the latest version

**Pull request merged**